charts/lfx-platform/templates/openfga/model.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{- if and .Values.openfga.enabled (index .Values "fga-operator" "enabled") }}
+apiVersion: extensions.fga-operator/v1
+kind: AuthorizationModelRequest
+metadata:
+  name: {{ index .Values "fga-operator" "store" }}
+  labels:
+    {{- include "lfx-platform.labels" . | nindent 4 }}
+    app.kubernetes.io/component: openfga
+spec:
+  instances:
+    - version:
+        major: 1
+        minor: 1
+        patch: 1
+      authorizationModel: |
+        model
+          schema 1.1
+
+        type user
+
+        type team
+          relations
+            define member: [user]
+
+        type project
+          relations
+            define parent: [project]
+            define owner: [team#member] or owner from parent
+            define writer: [user] or owner or writer from parent
+            define auditor: [user, team#member] or writer or auditor from parent
+            define viewer: [user:*] or auditor or auditor from parent
+
+        type committee
+          relations
+            define member: [user]
+            define project: [project]
+            define writer: writer from project
```

Comment:
this is fine, but I wanted to call out that we are already talking about having this be a field in the new model, which would make this `[user] or writer from project` ... HOWEVER leaving it like this for now is an excuse to test out having the operator push a change for us! :-)

Reply:
I played around with updating models today, and it worked well! You merely need to increment the `version` and modify the model. Alternatively we may want to always keep the current and previous model here, and add another instance when we make a change.